.dockerignore

```diff
@@ -1,3 +1,4 @@
 .env
 .deploy-env
-iasql-on-iasql.out.sql
\ No newline at end of file
+iasql-on-iasql.out.sql
+node_modules
```

Comment:
I had to add this since I was getting this error 🤔 

<img width="903" alt="image" src="https://user-images.githubusercontent.com/5898639/186743416-5f1e1bf2-c048-4b6d-84ab-19c5e22b99be.png">